client/src/components/CardList/CardList.stories.tsx

```diff
@@ -27,7 +31,7 @@ const elements = [
       { display: "Href 3", href: "#" },
     ],
   },
-];
+] as unknown as CardListElementProps[];
```

Comment:
To be clear, with `"strictBindCallApply": true` none of this will be necessary. `Basic.args` will know its own types, so `elements` will infer that it should be `CardListElementProps[]`.

Reply:
I forgot that having href with empty string would have the same effect as having no href for the Non-link Title elements